code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -28,6 +28,8 @@
 	var/skipeyes = skipitems & HIDEEYES
 	var/skipears = skipitems & HIDEEARS
 
+	var/list/visible_slots = list(w_uniform, head, wear_suit, s_store, back, l_hand, r_hand, gloves, belt, shoes, wear_mask, l_ear, r_ear, wear_id)
```

Comment:
Some of these are good in theory but just wouldn't match the message below. A better way of handling it would be "They are looking at their [thing]" or similar.

Reply:
That's actually much better. I was looking for better wording, but was struggling.